src/components/Members.jsx

```diff
@@ -32,14 +35,16 @@ function Members() {
       </div>
       <div className="members"> 
         <h3>Meet the Contributors</h3>
-        <div className="row">
+        <a className="contributors-link" href="https://github.com/FCCColumbus/cbus-web/graphs/contributors">
+        <div className="row">          
           {members.map((member) =>
             <div key={member.id}>
               <img className='avatar member-avatar' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
-              <h4>{member.name}</h4>
+              <h5>{member.name}</h5>
```

Comment:
Is there a way to change the spacing on these without changing the semantic structure? https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#best-practices-summarized